woocommerce/woocommerce-android
# [Login] List non-Woo stores in the site picker to allow installing Woo

Closes: #7129 

### Description
This PR updates the site picker during login to list non-woo sites, and allow installing Woo from the app, when tapping a non-woo site, the flow continues to the flow implemented in #7090.

### Testing instructions
1. Make sure one or more sites connected to your account don't have Woo.
2. Open the app, and sign in using your WordPress.com account.
3. In the site, confirm that the list is grouped to two sections: one for Woo stores, and one for other sites.
4. Click on one of the other sites.
5. Confirm the app opens the "Woo not found" error screen.
6. Click on "Install Woo"
7. Confirm the login continues successfully after installing Woo.

Feel free to repeat the test steps with different combinations of sites to confirm the flow works for all of them.

### Images/gif
##### Installation Flow
https://user-images.githubusercontent.com/1657201/183702903-8cbb7677-96cf-48c3-bfc7-a8f52b2ffaa4.mov

##### Single connected site is non-woo
<img width=360 src="https://user-images.githubusercontent.com/1657201/183872526-e20ce29b-1d99-4bb7-9d6a-80929c2b36f7.png"/>

##### Site picker for switching store
<img width=360 src="https://user-images.githubusercontent.com/1657201/183874535-b9da7b45-992a-4c7c-8f38-25b3311f127d.png"/>
